src/SDKs/Resource/Management.ResourceManager/Generated/SdkInfo_FeatureClient.cs

```diff
@@ -0,0 +1,18 @@
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+internal static partial class SdkInfo
```

Comment:
What are all these SdkInfo* classes? @shahabhijeet

Reply:
@vivsriaus these are generated to get more meta data about the underlying API version